app/fissile.go

```diff
@@ -951,6 +951,14 @@ func (f *Fissile) generateHelmValues(outputDir string, rolesManifest *model.Role
 		_, err = outputFile.Write(buf)
 	}
 	if err == nil {
+		_, err = outputFile.Write([]byte(`kube:
```

Comment:
I'm unable to come up with any way that is remotely readable and also acceptable to the Go compiler.  Feel free to attach exact code (that has passed the formatter and compiler).

I think Go is simply not a suitable language for dynamic data structures like this.